tests/Unit/Helpers/NumericalAlgorithms/SphericalHarmonics/YlmTestFunctions.hpp

```diff
@@ -9,11 +9,68 @@
 #include <vector>
 
 #include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "NumericalAlgorithms/Spectral/LogicalCoordinates.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
 #include "NumericalAlgorithms/SphericalHarmonics/Spherepack.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/SpherepackIterator.hpp"
 #include "Utilities/Gsl.hpp"
 
 namespace YlmTestFunctions {
 
+template <size_t l, int m>
+class Ylm {
+ public:
+  Ylm(size_t n_r, size_t L, size_t M);
+  DataVector f() const;
+  DataVector df_dth() const;
+  // This is the Pfaffiaan derivative (extra factor 1/sin(th))
```

Comment:
what do you mean `extra`? I assume you mean that this returns `(1/s(t)) df_dph`? Can you clarify the C++ comment?